### ISSUE TITLE
Changelog preview

### ISSUE THREAD

**Issue by lekru031** [d4rken/sdmaid-public#431]
Have you considered to display changelog like this, so when tapping changelog it opens a window instead of opening the browser everytime. It feels more convenient. 

![screenshot_2016-07-01-16-37-31](https://cloud.githubusercontent.com/assets/3966846/16542299/43b48980-40a3-11e6-8e7f-39099670ece0.png)

**Comment by d4rken** [comment]
Actually I already went through quite a few approaches to this 😄 .
In SD Maid v1 & v2 we had something like the dialog on your screenshot. That was okay, but larger texts were not very comfortable to read on it. As SD Maid grew it become too crowded. Some users also expressed dislike to the changelog always popping up on updates.

In SD Maid v3 I went with a small side bar that when clicked opened a new window with a full screen changelog. That was better as it wasn't "in your face" on each update and allowed more space for text and changes. Some users then asked to specifically move the changelog online to make SD Maid size smaller. Size wasn't an issue though as it only came down to a few KB (for all changes for all versions).

In SD Maid v4 I went with an online changelog for two main reasons:
* I only have to maintain the changelog in one location. This ties into the paradigm that a changelog makes the most sense when it can be viewed before updating an app.
* I can correct typos, add stuff I forgot or add further explanation for changes after publishing an APK.

This can work, but I'm happy with the current solution.
I have thought about displaying the changelog in a webview, but this seems superfluos, users have their favorite browsers, why not let them use those to view the changelog.
